Release Kaptain 1.2.0-1.1.0 (#3575)
mesosphere/dcos-docs-site: pages/dkp/kaptain/1.2.0-1.1.0/release-notes/1.1/index.md

---
layout: layout.pug
navigationTitle:  Release Notes for Kaptain 1.2.0-1.1.0
title: Release Notes for Kaptain 1.2.0-1.1.0
menuWeight: 5
beta: false
excerpt: View release-specific information for Kaptain
---

# Release Notes

## 1.2.0-1.1.0 - Released 12 May 2021

This document describes the new features, caveats, and resolved issues in D2iQ Kaptain.

### New features

> * Upgradability and configuration updates of existing Kaptain installations without user workloads interruption

Reviewer: ```suggestion
* Full zero-downtime upgradability of Kaptain and all of its configurations
```

Reviewer: The top feature should be a bit punchier (as long as it's accurate).
